README.md

```diff
@@ -6,86 +6,15 @@ If you are interested in the current state of planning, have a look at our [Rele
-- __Backlog:__ If enough information is gathered, and we agreed to work on the issue, it is set from `Inbox` to `Backlog` to indicate it is ready for timeline planning
-- __Work in Progress:__ The issue is actively been worked on.
-- __Done:__ All relevant parts have been implemented and released
-
-### Issue process
-
-Every new feature proposal or bug report will be handled as issue in status `Inbox` initially. The alignment meetings are used to discuss the purpose and impact of the current issues.
-While in `Inbox` status, the involved components are discovered and respective `Labels` are added. If already possible, a desired `Milestone` can be set.
-Additionally, an `Assignee` is selected, who will coordinate efforts to solve the issue.
-
-After these details are clarified, an issue is moved to `Backlog` to open it for detailed timeline planning. In this status, discussions about a fitting `Iteration` is held.
-
-As soon as actual work is started in the selected iteration, the issue is set to `Work in Progress`. This is especially helpful on our project milestone views to get an overview of the release progress.
-
-The final status `Done` is set, as soon as all relevant implementations are done, tested and released. This has to be achieved for every change in every involved component.
-
-### Planning component changes
-
-While the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26) is used to coordinate overarching feature and bug requests,
-we encourage every component team to break these issues down to their component repositories/projects.
-When doing so, make sure you link to the overarching issue in your component issue description.
-
-## Release Management Acceptance Criteria
-
-The release participation can be initiated by creating issues for the acceptance criteria check from the Issue templates.
-Each release the templates for the acceptance criteria will be renewed. There are two paths for processing the acceptance criteria for an application team to participate in a release.
-
-### The Release Happy Path for the Acceptance Criteria
-
-The three process steps to get to the status you need to pass the Q-Gate are shown in the happy path process flow.
-
-Each acceptance criteria issue in GitHub contains a note with the prime contacts so that it is clear who is the assigned expert or release manager.
-
-![Happy Path](docs/static/releasemanagement-acceptance-happy-path.svg)
-
-### The other Release Path
-
-If the evidence is not sufficient so that the criterium can not be accepted in the quality gate (QG), obligations for the product team will be defined to make a reassessment.
-![The other Path](docs/static/releasemanagement-acceptance-other-path.svg)
+See [the product release documentation](./docs/product_release.md), for information about singular product releases.
```

Comment:
could look nicer like this:
See [product release](./docs/product_release.md), for information about singular product releases.